react_projects/rick-and-morty-web/src/components/rawItem/RawItem.css

```diff
@@ -0,0 +1,54 @@
+.main{
+  display: flex;
+
+  margin-bottom: 1rem;
+
+  border: 1px solid #ccc; /* Default border */
+  transition: border 0.1s ease; /* Smooth transition for the border */
+}
+
+.main:hover {
+  border: 2px solid #007BFF; /* Change the border on hover */
+}
+
+.right {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+
+  padding: 1rem;
+}
+
+.item-image{
+  width: 7rem;
+  height: 7rem;
+}
+
+.name {
+  text-align: left;
+  width: auto;
+
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.description{
+  margin-top: 0.5rem;
+
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: flex-start;
+
+  font-size: 15px;
+}
+
+p {
```

Comment:
you have to be careful with these global styles. Once the CSS file is loaded in the browser, all the paragraphs are gonna take these styles.
At least this happens when using normal stylesheets